### PR TITLE
Added GitHub3.css

### PR DESCRIPTION
**Pull request by olvlvl**
Hi,

This commit provides a CSS that better match GitHub latest look.

### CODE REVIEW

**Comment by uranusjr** [comment]
Wow, thanks! This has been on my mind for quite some time, but I just couldn’t find the time to implement this myself.

The theme/style system is currently under some heavy refactoring (on my machine), so I’m not going to merge this immediately, but will wait until the refactoring is done. And after that I will need to tweak this a little so it can be compatible with the new system. But it should be ready with version 0.5.

**Comment by olvlvl** [comment]
Thanks for your reply. There's no rush for me since I can use the style file easily. If you need help to refactor this for the v0.5, you can count on me.

**Comment by AntonioMeireles** [comment]
:+1:

**Comment by zpbx** [comment]
@olvlvl 
Just a reminder: Structure of code blocks generated by MacDown is different from GitHub. So all these syntax highlight style rules (`.highlight *`) from GitHub's CSS are unnecessary, they don't take effect in MacDown.

@uranusjr 
BTW, do we have a plan to allow user to customize syntax highlight themes?

**Comment by olvlvl** [comment]
@zpbx Thanks for your comment. Should I remove the rules or change their selectors?

**Comment by zpbx** [comment]
@olvlvl 
I think it's fine to simply remove these syntax highlight styles for this PR. When @uranusjr open the entrance to customize syntax highlight themes, we can add GitHub's syntax highlight theme later -- that'll be more easy and proper.

**Comment by olvlvl** [comment]
Done!

**Comment by zpbx** [comment]
:+1:

**Comment by petems** [comment]
:+1: Looking forward to the merge :smile:

**Comment by frdmn** [comment]
Looks great. Can't wait for this in an official release :+1:

**Comment by KrisSiegel** [comment]
Anything ever happen to this?

**Comment by olvlvl** [comment]
Looks like it's heading to release 0.6. You can use the style sheet today if you want to:

1. Go to "Preferences / Rendering"
2. Click on "Reveal", this should open a Finder window.
3. Go in the "Styles" directory
4. Save the [style sheet](https://raw.githubusercontent.com/olvlvl/macdown/styles_github3/MacDown/Resources/Styles/GitHub3.css) there as "Github3.css".
5. Go back to MacDown and click "Reload", the style sheet should appear in the dropdown menu.
6. Select the style sheet.

Have fun!

**Comment by petems** [comment]
@olvlvl :love: Awesome! :smile:

**Comment by zvving** [comment]
:+1:

**Comment by chibicode** [comment]
I generated one out of [sindresorhus/github-markdown-css](https://github.com/sindresorhus/github-markdown-css), which is more up-to-date. Here it is:

https://github.com/chibicode/macdown-github-style

**Comment by olvlvl** [comment]
@JackoPlane What's up with this PR? It's been two years, I totally forgot about it.

**Comment by franklinyu** [comment]
@olvlvl I guess he just randomly jumped in. GitHub seems to have updated their style again... I'm looking for a way to automate this.

**Comment by franklinyu** [comment]
As mentioned in sindresorhus/generate-github-markdown-css#4, we can automate this by compiling the Sass file

```sass
@import "github-syntax-light/lib/github-light.css";
@import "primer-markdown/index.scss";
```

which depends on two npm packages: `github-syntax-light` and `primer-markdown`. Compiling Sass is easy now that we already depends on Ruby (by using CocoaPods); however, I'm not sure whether we are able to install npm packages without Node.js. @uranusjr are you happy with Node.js as yet another dev-dependency? (Unlike Ruby, Node.js is not included in macOS by default.)

**Comment by zmnv** [comment]
why is blocked?

File: MacDown/Resources/Styles/GitHub3.css [hunk 1]
```diff
@@ -0,0 +1,458 @@
+body {
```
Comment (zmnv) [inline MacDown/Resources/Styles/GitHub3.css]: How about octicons?

```css
@font-face {
  font-family: octicons-link;
  src: url(data:font/woff;charset=utf-8;base64,d09GRgABAAAAAAZwABAAAAAACFQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABEU0lHAAAGaAAAAAgAAAAIAAAAAUdTVUIAAAZcAAAACgAAAAoAAQAAT1MvMgAAAyQAAABJAAAAYFYEU3RjbWFwAAADcAAAAEUAAACAAJThvmN2dCAAAATkAAAABAAAAAQAAAAAZnBnbQAAA7gAAACyAAABCUM+8IhnYXNwAAAGTAAAABAAAAAQABoAI2dseWYAAAFsAAABPAAAAZwcEq9taGVhZAAAAsgAAAA0AAAANgh4a91oaGVhAAADCAAAABoAAAAkCA8DRGhtdHgAAAL8AAAADAAAAAwGAACfbG9jYQAAAsAAAAAIAAAACABiATBtYXhwAAACqAAAABgAAAAgAA8ASm5hbWUAAAToAAABQgAAAlXu73sOcG9zdAAABiwAAAAeAAAAME3QpOBwcmVwAAAEbAAAAHYAAAB/aFGpk3jaTY6xa8JAGMW/O62BDi0tJLYQincXEypYIiGJjSgHniQ6umTsUEyLm5BV6NDBP8Tpts6F0v+k/0an2i+itHDw3v2+9+DBKTzsJNnWJNTgHEy4BgG3EMI9DCEDOGEXzDADU5hBKMIgNPZqoD3SilVaXZCER3/I7AtxEJLtzzuZfI+VVkprxTlXShWKb3TBecG11rwoNlmmn1P2WYcJczl32etSpKnziC7lQyWe1smVPy/Lt7Kc+0vWY/gAgIIEqAN9we0pwKXreiMasxvabDQMM4riO+qxM2ogwDGOZTXxwxDiycQIcoYFBLj5K3EIaSctAq2kTYiw+ymhce7vwM9jSqO8JyVd5RH9gyTt2+J/yUmYlIR0s04n6+7Vm1ozezUeLEaUjhaDSuXHwVRgvLJn1tQ7xiuVv/ocTRF42mNgZGBgYGbwZOBiAAFGJBIMAAizAFoAAABiAGIAznjaY2BkYGAA4in8zwXi+W2+MjCzMIDApSwvXzC97Z4Ig8N/BxYGZgcgl52BCSQKAA3jCV8CAABfAAAAAAQAAEB42mNgZGBg4f3vACQZQABIMjKgAmYAKEgBXgAAeNpjYGY6wTiBgZWBg2kmUxoDA4MPhGZMYzBi1AHygVLYQUCaawqDA4PChxhmh/8ODDEsvAwHgMKMIDnGL0x7gJQCAwMAJd4MFwAAAHjaY2BgYGaA4DAGRgYQkAHyGMF8NgYrIM3JIAGVYYDT+AEjAwuDFpBmA9KMDEwMCh9i/v8H8sH0/4dQc1iAmAkALaUKLgAAAHjaTY9LDsIgEIbtgqHUPpDi3gPoBVyRTmTddOmqTXThEXqrob2gQ1FjwpDvfwCBdmdXC5AVKFu3e5MfNFJ29KTQT48Ob9/lqYwOGZxeUelN2U2R6+cArgtCJpauW7UQBqnFkUsjAY/kOU1cP+DAgvxwn1chZDwUbd6CFimGXwzwF6tPbFIcjEl+vvmM/byA48e6tWrKArm4ZJlCbdsrxksL1AwWn/yBSJKpYbq8AXaaTb8AAHja28jAwOC00ZrBeQNDQOWO//sdBBgYGRiYWYAEELEwMTE4uzo5Zzo5b2BxdnFOcALxNjA6b2ByTswC8jYwg0VlNuoCTWAMqNzMzsoK1rEhNqByEyerg5PMJlYuVueETKcd/89uBpnpvIEVomeHLoMsAAe1Id4AAAAAAAB42oWQT07CQBTGv0JBhagk7HQzKxca2sJCE1hDt4QF+9JOS0nbaaYDCQfwCJ7Au3AHj+LO13FMmm6cl7785vven0kBjHCBhfpYuNa5Ph1c0e2Xu3jEvWG7UdPDLZ4N92nOm+EBXuAbHmIMSRMs+4aUEd4Nd3CHD8NdvOLTsA2GL8M9PODbcL+hD7C1xoaHeLJSEao0FEW14ckxC+TU8TxvsY6X0eLPmRhry2WVioLpkrbp84LLQPGI7c6sOiUzpWIWS5GzlSgUzzLBSikOPFTOXqly7rqx0Z1Q5BAIoZBSFihQYQOOBEdkCOgXTOHA07HAGjGWiIjaPZNW13/+lm6S9FT7rLHFJ6fQbkATOG1j2OFMucKJJsxIVfQORl+9Jyda6Sl1dUYhSCm1dyClfoeDve4qMYdLEbfqHf3O/AdDumsjAAB42mNgYoAAZQYjBmyAGYQZmdhL8zLdDEydARfoAqIAAAABAAMABwAKABMAB///AA8AAQAAAAAAAAAAAAAAAAABAAAAAA==) format('woff');
}
```

**Comment by zmnv** [comment]
Will it be release in official version?

**Comment by franklinyu** [comment]
@zmnv Working on it, but no ETA now.

**Pull request closed without merging**